doc/sphinx/source/index.rst

```diff
@@ -1,10 +1,10 @@
 =============================
-KHIVA (Time Series Analytics)
+Khiva (Time Series Analytics)
```

Comment:
Probably, we could delete (Time Series Analytics), as it doesn´t make sense anymore.